ui/widgets/nuxeo-selectivity.js

```diff
@@ -7214,16 +7214,46 @@ typedArrayTags[weakMapTag] = false;
 
       // scope styles
       this.scopeSubtree(this.$.input, true);
+
+      // use observer to hide dropdown if not visible in the viewport
+      this._visibilityObserver = new IntersectionObserver((entries) => {
+        if (entries && entries.length > 0) {
+          const entry = entries[0];  // there should be only one entry here
+          if (!entry.isVisible) {
+            this._selectivity.close();
+          }
+        }
+      }, {
+        rootMargin: '0px',
+        threshold: 1.0
+      });
+      this._visibilityObserver.observe(this);
+
+      // attach event listener to scroller parent to hide dropdown
+      this._scrollParent = this._getScrollParent(this);
+      if (this._scrollParent) {
+        this._scrollParent.addEventListener('scroll', this._closeSelectivityDropdown.bind(this));
```

Comment:
With this in place, do we still need the IntersectionObserver?

Reply:
No, we don't need the `IntersectionObserver` if we go with the scrollable parent approach. I kept both solutions here to spark the discussion.

Reply:
Aah, sorry. Thank you. Can you please add that to the description of the PR? Didn't notice we had two alternatives here.

Reply:
Just removed the use of the IntersectionObserver, since we're going with the scrollable parent solution.